lighthouse-core/test/report/v2/renderer/report-renderer-test.js

```diff
@@ -103,6 +103,15 @@ describe('ReportRenderer V2', () => {
       });
     });
 
+    it('renders a warning when using headless chrome', () => {
+      const container = renderer._dom._document.body;
+      const reportJson = Object.assign(sampleResults, {
+        userAgent: 'HeadlessChrome/62.0.3180.0 Safari/537.36'
+      });
+      const output = renderer.renderReport(reportJson, container);
```

Comment:
this is changing the contents of the container, which is going to affect other tests in this file since setup/teardown is only done once. Should `before`/`after` above actually be `beforeEach`/`afterEach`?